crates/languages/src/rust/config.toml

```
@@ -12,3 +12,6 @@ brackets = [
     { start = "/*", end = " */", close = true, newline = false, not_in = ["string", "comment"] },
 ]
 collapsed_placeholder = " /* ... */ "
+test_templates = [
+    {capture_name = "cargo-test", label = "cargo test", template = "cargo test -p ZED_PACKAGE ZED_TEST_CAPTURES[0]"}
```

```suggestion
    {capture_name = "cargo-test", label = "cargo test", template = "cargo test -p $ZED_PACKAGE $ZED_TEST_CAPTURES[0]"}
```
these variables are env variables proper.
I wonder if `shellexpand` supports arrays 🤔 We use that library to expand uses of Zed environment variables in labels. If it does support it, we could put the name of a test in a label and the task system will expand it for us.

---

Ok, I've checked with shellexpand and we can use it to substitute array subscripts just fine, e.g. during parsing of the command we'll come across `ZED_TEST_CAPTURES[0]`; we have to parse the subscript part ourselves, but that shouldn't be too hard :)

---

This was discussed further in a pairing session, @osiewicz and I agreed that putting the tasks in the config file is unnecessary, we can just expose the test capture names and make a linking in the existing tasks.json file.